src/utils.cxx

```diff
@@ -154,9 +155,41 @@ std::string getCmdOption(char ** begin, char ** end, const std::string & option)
     return 0;
 }
 
+void StripPathAndExe(std::string &command) {
+    StripPath(command);
+    StripExe(command);
+};
+
+void StripExe(std::string &command) {
+    // Normalize command to lowercase to avoid parsing issues
+    std::transform(command.begin(), command.end(), command.begin(),
+        [](unsigned char c){ return std::tolower(c); });
+    std::string::size_type loc = command.rfind(".exe");
+    if ( std::string::npos != loc )
+        command.erase(loc);
+}
+
+void StripPath(std::string &command) {
+    command.erase(0, command.find_last_of("\\/") + 1);
+}
+
+
+bool CLICheck(const char * arg, const char * check)
+{
+    std::string normalized_arg(arg);
+    StripPathAndExe(normalized_arg);
+    return strcmp(normalized_arg.c_str(), check) == 0;
+}
+
+
 bool IsRelocate(const char * arg)
```

Comment:
For logic related to parsing cmd line args for the wrapper. I think `utils` is too generic a spot for them: IMO if they should exist outisde of `main` then I think a command-line parsing module would be a good spot for them.

Reply:
Makes sense!